nn/conformer.py

```diff
@@ -37,16 +37,17 @@ def forward(self, inp: nn.LayerRef) -> nn.LayerRef:
     return x_ff2
 
 
-class _ConformerConvBlock(nn.Module):
+class ConformerConvBlock(nn.Module):
   """
   Conformer convolution block
       FF -> GLU -> depthwise conv -> BN -> Swish -> FF
   """
 
   def __init__(self, out_dim: int, kernel_size: int, batch_norm_opts: Optional[Dict[str, Any]] = None):
     """
-    :param out_dim:
-    :param kernel_size:
+    :param out_dim: output feature dimension
+    :param kernel_size: kernel size of depthwise convolution
+    :param batch_norm_opts: batch norm options
```

Comment:
I really don't like descriptions which just basically repeat what the argument name already says...